openverse_catalog/dags/data_refresh/refresh_view_data_task_factory.py

```diff
@@ -0,0 +1,39 @@
+"""
+# Refresh Materialized View Task Factory
+This file generates a Task that refreshes the materialized view for a
+given media type, using a factory function.
+
+The task refreshes the materialized view, but not the underlying tables. This
+means that the only effect is to add or update data (including popularity data)
+for records which have been ingested since the last time the view was
+refreshed.
+
+This should be run every time before a data refresh is triggered.
+"""
+import os
+
+from airflow.utils.trigger_rule import TriggerRule
+from common.popularity import operators
+
+
+DB_CONN_ID = os.getenv("OPENLEDGER_CONN_ID", "postgres_openledger_testing")
```

Comment:
I feel like we should centralize this somewhere 🤔 maybe in `constants.py`? It seems annoying to have to grab it from the env so frequently.